backend/de.metas.acct.webui/src/main/java/de/metas/acct/gljournal_sap/quickinput/SAPGLJournalLineQuickInputDescriptorFactory.java

```diff
@@ -39,8 +43,10 @@
 @RequiredArgsConstructor
 public class SAPGLJournalLineQuickInputDescriptorFactory implements IQuickInputDescriptorFactory
 {
+	@NonNull private static final String SYS_CONFIG_FilterAccount = "de.metas.acct.gljournal_sap.quickinput.SAPGLJournalLineQuickInputDescriptorFactory.AccountValRuleId";
```

Comment:
why do we need a sysconfig for that and not simply hardcode it to `540695`?
what's the actual real world benefit?

Reply:
or more, if u really want to have it configurable then why not getting POInfo.get(gl journal line).getValRuleId(column name)?